rosa_release_notes/rosa-release-notes.adoc

```diff
@@ -0,0 +1,104 @@
+:_content-type: REFERENCE
+[id="rosa-release-notes"]
+= {product-title} 4.10 release notes
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: rosa-release-notes
+
+toc::[]
+
+{product-title} (ROSA) is a fully-managed, turnkey application platform that allows you to focus on delivering value to your customers by building and deploying applications. Red Hat and AWS site reliability engineering (SRE) experts manage the underlying platform so you do not have to worry about the complexity of infrastructure management. ROSA provides seamless integration with a wide range of AWS compute, database, analytics, machine learning, networking, mobile, and other services to further accelerate the building and delivering of differentiating experiences to your customers.
+
+[id="about-this-release-section_{context}"]
+== About this release
+
+{product-title} https://access.redhat.com/errata/RHEA-2022:2252[RHEA-2022:2252] is now available. This release uses Kubernetes 1.24 with CRI-O runtime. New features, changes, and known issues that pertain to {product-title} 4.10 are included in this topic.
+
+{product-title} clusters are available at https://console.redhat.com/openshift. With the {cluster-manager-first} application for ROSA, you can deploy {product-title} clusters to either on-premise or cloud environments.
+
+[id="about-this-release_{context}"]
+== ROSA layered and dependent component support and compatibility
+The scope of support for layered and dependent components of ROSA changes independently of the ROSA version. To determine the current support status and compatibility for an add-on, refer to its release notes. For more information, see the xref:../rosa_architecture/rosa_policy_service_definition/rosa-life-cycle.adoc#rosa-life-cycle[{product-title} update life cycle].
```

Comment:
"To determine the current support status and compatibility for an add-on, refer to its release notes."

This sentence confused me a little. Were are people supposed to refer to? is it the "[Red Hat OpenShift Service on AWS update life cycle]" link?

Reply:
Good catch, and you're right. Updated the text.